mealie/core/config.py

```diff
@@ -151,6 +151,11 @@ def public_db_url(cls, v: Optional[str], values: dict[str, Any]) -> str:
     DEFAULT_EMAIL: str = "changeme@email.com"
     DEFAULT_PASSWORD: str = "MyPassword"
 
+    USE_LDAP_AUTH: bool = False
+    LDAP_SERVER_URL: str = None
+    LDAP_BIND_TEMPLATE: str = None
+    LDAP_ADMIN_FILTER: str = None
```

Comment:
Can you document these new variables in the `template.env` file and the list of env variables in the docs? 

https://github.com/hay-kot/mealie/blob/master/docs/docs/documentation/getting-started/install.md